srcopsmetrics/cli.py

```diff
@@ -98,6 +99,12 @@ def get_entities_as_list(entities_raw: Optional[str]) -> List[str]:
             are stored. Default knowledge path is {StoragePath.DEFAULT.value}
             """,
 )
+@click.option(
+    "--metrics", "-m", is_flag=True, required=False, help=f"""Launch Metrics Calculation for specified repository.""",
+)
+@click.option(
+    "--kebechet", "-K", is_flag=True, required=False, help=f"""Launch Metrics Calculation for specified repository.""",
```

Comment:
the help needs to be corrected, as it is the same as for `metrics` option

Reply:
corrected